js/currencyTable.js

```diff
@@ -0,0 +1,105 @@
+const CurrencyTable = function ( global ) {
+    
+    let investAmount    = 0;
+    let allCurrencies   = [];
+    let fiat            = ['$', 'USD'];
+    const c20Index      = [ 13.4, 12.3, 10.5, 10.2, 9.2, 
+                            7.6, 5.4, 4.9, 4.3, 4, 
+                            3.8, 3.4, 2.1, 1.7, 1.5, 
+                            1.5, 1.2, 1.2, 0.9, 0.9 ];
+    // ===== News Interval ===== //
+    let newsInterval;
+
+
+    /**
+     * 
+     * @param {currencies} Array array of currencies data  
+     */
+    function buildTable( currencies ) {
+        // Add fiat to thead
+        document.querySelectorAll('.fiat').forEach((item) => item.textContent = ` (${fiat[1]})`);
+        
+        allCurrencies = currencies;
+        
+        let tbody = document.createElement( 'tbody' );
+        let currSymbols = [];
+        currencies.forEach( ( cur, coin ) => {
+            currSymbols.push(cur.symbol);
+            let tr = document.createElement( 'tr' );
+            tr.innerHTML = `
+                    <td>${ cur.rank }</td>
+                    <td>${ cur.name }</td>
+                    <td>${ formatNum( cur.market_cap_usd ) }</td>
+                    <td>${ formatNum( cur.price_usd ) }</td>
+                    <td>${ c20Index[coin] }%</td>
+                    <td>${ calculateInvestment( c20Index[coin] ) }</td>
+            `;
+            tbody.appendChild( tr )      
+        });
+        global.currencyTable.appendChild( tbody );
+        global.getNews(currSymbols);
+        if( newsInterval ) clearInterval( newsInterval );
```

Comment:
What are you doing here? Is this during the inital loading, if there a interval, clear it?